test/platform.ts

```diff
@@ -62,6 +67,11 @@ export interface IEmulatorManager {
      * Prepares the emulator for a test.
      */
     prepareEmulatorForTest(appId: string): Q.Promise<string>;
```

Comment:
The emulator manager is also the right place for [this](https://github.com/Microsoft/cordova-plugin-code-push/pull/101/files#diff-b9e12334e9eafd8341a6107dd98510c9R206).